includes/wizards/class-subscriptions-wizard.php

```diff
@@ -49,6 +42,33 @@ public function __construct() {
 		add_filter( 'woocommerce_product_data_store_cpt_get_products_query', [ $this, 'handle_only_get_newspack_subscriptions_query' ], 10, 2 );
 	}
 
+	/**
+	 * Get the name for this wizard.
+	 *
+	 * @return string The wizard name.
+	 */
+	public function get_name() {
+		return __( 'Subscriptions', 'newspack' );
```

Comment:
We should always use the escaping alternatives for localisation functions where possible to ensure that translations don't introduce any unsafe output. E.g. `esc_html__()`